packages/xchain-binance/src/client.ts

```diff
@@ -265,13 +241,13 @@ class Client implements BinanceClient, XChainClient {
   /**
    * Get the balance of a given address.
    *
-   * @param {Address} address By default, it will return the balance of the current wallet. (optional)
+   * @param {Address | number} address By default, it will return the balance of the current wallet. (optional)
    * @param {Asset} asset If not set, it will return all assets available. (optional)
    * @returns {Array<Balance>} The balance of the address.
    */
-  getBalance = async (address?: Address, assets?: Asset[]): Promise<Balances> => {
+  getBalance = async (address: Address, assets?: Asset[]): Promise<Balances> => {
```

Comment:
`address: string | number` is hard to understand. Try to introduce `index` (similar to one of previous messages)

Reply:
so here address must be a string(never a number), as discussed with @davidp94 we decided to allow getBalance(address) not be limited to only the addresses in your own wallet